### ISSUE TITLE
Druidia Road 1 Out of Fuel

### ISSUE THREAD

**Issue by sabres207** [anprogrammer/OpenRoads#3]
Hey guys, first of all, thanks for making this beautiful game online, and very good job.
I only had one problem with it - in the first road in druidia I always run out of fuel, no matter how I play it. I'm pretty sure that wasn't the case in original game.

Thanks.

**Comment by rishab96** [comment]
Yes I was playing recently and I'm certain this is a bug. I tried the same level on the original version of the game and it doesnt run out there. The fuel runs out just before it should.

**Comment by hanczakos** [comment]
I tried this on different speeds as well but I encountered the same problem. 
Though this article mentions something about the same level in the original game as well:
http://tasvideos.org/GameResources/DOS/Skyroads.html#2
Can we have some feedback on this?
